charts/k8ssandra-operator/templates/role.yaml

```diff
@@ -1,7 +1,8 @@
+{{- if not .Values.global.clusterScoped }}
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: {{ include "k8ssandra-common.fullname" . }}-k8ssandra-operator
```

Comment:
There's a few places where the suffix was removed like this... it makes things a lot easier to type and look at so I'm good with it, but just wanted to be sure it was intentional -- is this the "naming change" you mention in the PR description?

It does mean that copy and paste of examples in the current install guide no longer work -- we can clean that up as we migrate and write full docs though I think.

Reply:
The naming changes were intentional so as to be consistent with how we do naming elsewhere in the charts. 

I forgot about the install guide when I made the changes. I'll open a PR to update it.